interfaces/builtin/docker_support_test.go

```diff
@@ -26,33 +26,70 @@ import (
 	"github.com/snapcore/snapd/interfaces/apparmor"
 	"github.com/snapcore/snapd/interfaces/builtin"
 	"github.com/snapcore/snapd/interfaces/seccomp"
+	"github.com/snapcore/snapd/interfaces/udev"
 	"github.com/snapcore/snapd/release"
 	"github.com/snapcore/snapd/snap"
 	"github.com/snapcore/snapd/snap/snaptest"
 	"github.com/snapcore/snapd/testutil"
 )
 
 type DockerSupportInterfaceSuite struct {
-	iface    interfaces.Interface
-	slotInfo *snap.SlotInfo
-	slot     *interfaces.ConnectedSlot
-	plugInfo *snap.PlugInfo
-	plug     *interfaces.ConnectedPlug
+	iface                    interfaces.Interface
+	slotInfo                 *snap.SlotInfo
+	slot                     *interfaces.ConnectedSlot
+	plugInfo                 *snap.PlugInfo
+	plug                     *interfaces.ConnectedPlug
+	networkCtrlSlotInfo      *snap.SlotInfo
+	networkCtrlSlot          *interfaces.ConnectedSlot
+	networkCtrlPlugInfo      *snap.PlugInfo
+	networkCtrlPlug          *interfaces.ConnectedPlug
+	privContainersPlugInfo   *snap.PlugInfo
+	privContainersPlug       *interfaces.ConnectedPlug
+	noPrivContainersPlugInfo *snap.PlugInfo
+	noPrivContainersPlug     *interfaces.ConnectedPlug
 }
 
 const coreDockerSlotYaml = `name: core
 version: 0
 type: os
 slots:
   docker-support:
+  network-control:
 `
 
 const dockerSupportMockPlugSnapInfoYaml = `name: docker
 version: 1.0
 apps:
  app:
   command: foo
-  plugs: [docker-support]
+  plugs: 
+   - docker-support
+   - network-control
+`
+
+const dockerSupportPrivilegedContainersFalseMockPlugSnapInfoYaml = `name: docker
+version: 1.0
+plugs:
+ privileged:
+  interface: docker-support
```

Comment:
In the previous tests we used `privileged-containers: false`.

Reply:
Ah thanks for catching this. I added it back